var/spack/repos/builtin/packages/eckit/package.py

```diff
@@ -18,6 +18,8 @@ class Eckit(CMakePackage):
 
     maintainers("skosukhin", "climbfuji")
 
+    # To be replaced with 1.23.1 when tag is available
```

Comment:
i'm confused by this since 1.23.1 has been added here...not 1.23.2

but if i'm missing something (and most likely am) just say You Are missing the point and i will approve 😂

Reply:
No, my comment may be confusing. We called the fake release `1.23.1-pre`, because back then `1.23.0` was the latest release. ECMWF has since created `1.23.1`, but it did not contain our changes, therefore we should be using `1.23.2-pre` here

Reply:
thanks for clarifying :)!